ote/src/cljs/ote/style/front_page.cljs

```diff
@@ -17,30 +17,36 @@
                ::stylefy/media {{:max-width (str width-xl "px")} {:background      "url(/img/hero-2000.png)"}
                                 {:max-width (str width-l "px")}  {:background      "url(/img/hero-1600.png)"}
                                 {:max-width (str width-sm "px")} {:background      "url(/img/hero-1080.png)"
-                                                                  :height          "371px"}
+                                                                  :background-size "cover"
+                                                                  :height          "400px"}
                                 {:max-width (str width-xs "px")} {:background      "url(/img/hero-800.png)"
-                                                                  :height          "274px"}}})
+                                                                  :background-size "cover"
+                                                                  :height          "350px"}}})
 
 (def hero-btn {:padding-top "60px"
                ::stylefy/media {{:max-width (str width-xl "px")} {:padding-top "60px"}
                                 {:max-width (str width-l "px")} {:padding-top "60px"}
                                 {:max-width (str width-sm "px")} {:padding-top "60px"}
-                                {:max-width (str width-xs "px")} {:padding-top "20px"}}})
+                                {:max-width (str width-xs "px")} {:padding-top "40px"}}})
 
 (def h2 {:font-size "2.25em"
          ::stylefy/media {{:max-width (str width-xs "px")} {:font-size "1.5em"}}})
 
-(def fp-btn-blue {:background-image "linear-gradient(90deg, #06c, #0029b8)"})
+(def fp-btn-blue {:box-shadow "3px 3px 8px 0 rgba(0, 0, 0, .2)"
+                  :cursor "pointer"
+                  :background-image "linear-gradient(90deg, #06c, #0029b8)"})
+(def fp-btn-hover {::stylefy/mode {:hover {:box-shadow "1px 1px 4px 0 rgba(0, 0, 0, .2)"
+                                           :-webkit-transform "scale(0.98)"
```

Comment:
You can add autoprefixes using builtin garden-features like:
::stylefy/vendors ["webkit" "moz" "o"]
::stylefy/auto-prefix #{:transform}}